### PR TITLE
Preparing release qiskit-ibm-transpiler 0.6.0

### PR DESCRIPTION
**Pull request by jesus-talavera-ibm**
### Summary

Preparing release qiskit-ibm-transpiler 0.6.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +28,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
```
Comment (Eric-Arellano) [inline CHANGELOG.md:28]: Fwiw, I don't think you need to maintain this anymore thanks to release notes. Release notes cover the user-facing changes, and Git log covers the developer changes.

You could keep this file and point it to docs.quantum.ibm.com/api/qiskit-ibm-transpiler/release-notes (once it goes live)

Comment (jesus-talavera-ibm) [reply CHANGELOG.md:30]: OK, thank you @Eric-Arellano . I added that link in [CHANGELOG.md](https://github.com/Qiskit/qiskit-ibm-transpiler/pull/69/commits/3b0a265fb123e06bdb14dbf6fb46b3c0f72ad38e) so we don't need to update that file anymore

**Review by cbjuan — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -28,6 +30,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+
+### 🎉 Added
+
+- [#67](https://github.com/Qiskit/qiskit-ibm-transpiler/pull/67) Adds optimization_preferences as an option for TranspilerService
+- [#70](https://github.com/Qiskit/qiskit-ibm-transpiler/pull/70) Enabling coupling_map as options to the AI Synthesis passes
+
+### ✏️ Changed
+
+- [#63](https://github.com/Qiskit/qiskit-ibm-transpiler/pull/63) Fixing the type of ECR gates when loading circuits from the qiskit transpiler service
+- [#52](https://github.com/Qiskit/qiskit-ibm-transpiler/pull/52) Keep trying getting result even in case of HTTPError exception with status 520
```
Comment (Eric-Arellano) [inline CHANGELOG.md:43]: These should probably be in the release notes file 0.6.0.rst? They're bug fixes? 

You can manually add them like this:

https://github.com/Qiskit/qiskit-ibm-transpiler/blob/8c8168eff1b780cef9ead93a223e4f5069c6d459/release-notes/0.4.9.rst?plain=1#L4-L7

Comment (jesus-talavera-ibm) [reply CHANGELOG.md:43]: Sorry! I missed this last comment.
Here is the fix: https://github.com/Qiskit/qiskit-ibm-transpiler/pull/72

**Pull request merged**